dependencies.yaml

```diff
@@ -1141,6 +1144,17 @@ dependencies:
           - matrix:
             packages:
               - *rmm_unsuffixed
+  depends_on_rapids_logger:
+    common:
+      - output_types: [conda, requirements, pyproject]
+        packages:
+          - rapids-logger==0.1.*,>=0.0.0a0
+      - output_types: requirements
+        packages:
+          # pip recognizes the index as a global option for the requirements.txt file
+          # This index is needed for rapids_logger
+          - --extra-index-url=https://pypi.nvidia.com
```

Comment:
```suggestion
```

very minor comment: I don't think this "is needed for rapids_logger" comment is providing much value in a group that only has `rapids-logger` in it and that, by its name, only ever will.

less minor comment: are we planning to publish these to `pypi.nvidia.com`? I'd expect that we could skip that and just do `pypi.org` + the RAPIDS nightly index, like we did with `libucx-cu{11,12}` wheels. If I'm right about that, then `pypi.nvidia.com` isn't needed here.

Reply:
Good call. I think we will still be dual-publishing to pypi.nvidia.com because that's Kitmaker's policy, but as long as it's on pypi.org we can drop this index. We will need this on PyPI for cudf to be on PyPI anyway.

Reply:
Removed in https://github.com/rapidsai/cudf/pull/17899/commits/d2403baa3d4b3fb690b4699754db7c17a41e2641.

Reply:
> We will need this on PyPI for cudf to be on PyPI anyway.

ha oh yeah 😅